perfkitbenchmarker/spark_service.py

```diff
@@ -44,6 +44,9 @@
 RUNTIME = 'running_time'
 WAITING = 'pending_time'
 
+SPARK_JOB_TYPE = 'spark'
+HADOOP_JOB_TYPE = 'hadoop'
```

Comment:
No, it's not used in this file.  It's used in linux_benchmarks/spark_benchmark.py, linux_benchmarks/terasort_benchmark.py, providers/gcp/gcp_dataproc.py, and providers/aws/aws_emr.py.